### PR TITLE
Ying/add admin

### PR DESCRIPTION
**Pull request by yingtluo**
Gave current admins the ability to add new admin accounts if they wish to

image

image

image

image

image

### CODE REVIEW

**Comment by amillman** [comment]
can you put the error toast above the transparent black overlay? maybe lower the z-index of that black overlay or just make the z-index of all toasts really high to put them on top of everything

File: app/assets/javascripts/components/admin/add_admin_modal.es6.jsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      this.resetPassword()
+    }
+  }
+
+  render() {
+    return(
+      <Modal show={true} onHide={this.props.handleClose}>
+
+        <Modal.Header closeButton>
+          <Modal.Title>Make Someone an Admin</Modal.Title>
```
Comment (amillman) [inline app/assets/javascripts/components/admin/add_admin_modal.es6.jsx]: "Invite an Admin"

File: app/assets/stylesheets/components/_popovers.scss [hunk 2]
```diff
@@ -21,10 +21,12 @@
   &.toast-error {
     background: $red;
     color: white;
+    z-index: 10000;
```
Comment (echang9) [inline app/assets/stylesheets/components/_popovers.scss]: hmm does the z-index need to be this large? :o

File: app/assets/stylesheets/components/_popovers.scss [hunk 3]
```diff
@@ -21,10 +21,12 @@
...
   }
 
   &.toast-success {
     background: #2ecc71;
     color: white;
+    z-index: 10000;
```
Comment (echang9) [inline app/assets/stylesheets/components/_popovers.scss]: same here

File: app/assets/javascripts/components/admin/add_admin_modal.es6.jsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    };
+  }
+
+  static get propTypes() {
+    return {
+      handleClose: React.PropTypes.func,
+    };
+  }
+
+  renderError() {
```
Comment (echang9) [inline app/assets/javascripts/components/admin/add_admin_modal.es6.jsx]: do we need to renderError if there's a toastr?

**Pull request merged**